plugins/scanners/scanoss/src/test/kotlin/ScanOssScannerDirectoryTest.kt

```diff
@@ -110,4 +118,61 @@ class ScanOssScannerDirectoryTest : StringSpec({
             )
         }
     }
+
+    "Scanner should exclude only files matching the specified path pattern (**/*.kt)" {
+        val pathExcludes = listOf(
+            PathExclude(
+                pattern = "**/*.kt", // Glob pattern to match all .kt files in any directory
+                reason = PathExcludeReason.BUILD_TOOL_OF,
+                comment = "Excluding .kt source files from scanning"
+            )
+        )
+
+        // Verify our test file exists. This file should be included in the scan since it doesn't match the exclusion
+        // pattern (it's a .go file, not .kt).
+        val includedFile = File(EXCLUSION_TEST_DIRECTORY, "server.go")
+        if (!includedFile.exists()) {
+            fail("file ${includedFile.absolutePath} does not exist")
+        }
+
+        // Run the scanner with our exclusion pattern. This will traverse the directory and should skip .kt files
+        scanner.scanPath(
+            EXCLUSION_TEST_DIRECTORY,
+            ScanContext(
+                labels = emptyMap(),
+                packageType = PackageType.PACKAGE,
+                excludes = Excludes(paths = pathExcludes)
+            )
+        )
+
+        // Retrieve all HTTP POST requests captured by WireMock during the scan
+        val requests = server.findAll(WireMock.postRequestedFor(WireMock.anyUrl()))
+        val requestBodies = requests.map { it.bodyAsString }
+
+        // The scanner sends files to the API in a multipart/form-data POST request with this format:
+        // --boundary
+        // Content-Disposition: form-data; name="file"; filename="[UUID].wfp"
+        // Content-Type: text/plain; charset=utf-8
+        // Content-Length: [length]
+        //
+        // file=[hash],[size],[filename]  <-- This is the format we're extracting with regex
```

Comment:
Avoid "we" etc. and use passive voice. Like "This is the format extracted by the regex".